Python/Setup/swix/swix.generate.targets

```diff
@@ -49,10 +49,6 @@ package name=$([System.IO.Path]::GetFileNameWithoutExtension($(OutputName))) ver
 <WriteLinesToFile File="$(SwrFile).tmp" Lines="vs.dependencies
 $(_Dependencies)
 " Overwrite="no" Encoding="ASCII" Condition="$(_Dependencies) != ''" />
-
-    <ItemGroup>
-      <Package Include="$(SwrFile)" />
-    </ItemGroup>
   </Target>
 
   <Target Name="_GenerateSwrFromSources" Condition="@(SwrSource) != ''">
```

Comment:
If you write out the list of every file into a static swr (and keep it up to date over time) then you don't need any of this. I was too lazy to do that :)

---
In reply to: [69642273](https://github.com/Microsoft/PTVS/pull/1396#discussion_r69642273) [](ancestors = 69642273)